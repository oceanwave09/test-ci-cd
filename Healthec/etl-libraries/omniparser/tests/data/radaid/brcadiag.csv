batch_type,First Name,Last Name,MRN/ID,DIAG_DATE,DIAG_NAME,DIAG_CODE,DIAG_RESULT
4,First26351,Last26351,26351,10-15-2023,1,77062,2
4,First26352,Last26352,26352,09-18-2014,2,G0279,3
4,First26353,Last26353,26353,12-15-2001,3,77065,6