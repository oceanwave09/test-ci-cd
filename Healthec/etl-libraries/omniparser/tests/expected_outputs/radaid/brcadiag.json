[
	{
		"batch_type": "4",
		"diag_code": "77062",
		"diag_date": "10-15-2023",
		"diag_name": "1",
		"diag_result": "2",
		"first_name": "9e7a7d800f451bef2a6bb9e2024d7569",
		"last_name": "74967d45b9d2cfa01bd08352d5ae7802",
		"mrn": "1c26ac8bfc15d08afbccac2ab1ec40fd",
		"row_id": "3999c7d1-f046-427b-85e8-f127ddd4164c"
	},
	{
		"batch_type": "4",
		"diag_code": "G0279",
		"diag_date": "09-18-2014",
		"diag_name": "2",
		"diag_result": "3",
		"first_name": "8f5a96dc34bd18e95f0fae1ef8328649",
		"last_name": "137fa153590188b32101b22447adbff0",
		"mrn": "3d7c0ad3e51ca2d9fa55d2c374bcd17a",
		"row_id": "c19b7d78-40fa-48d4-9047-8a6fc632fbd8"
	},
	{
		"batch_type": "4",
		"diag_code": "77065",
		"diag_date": "12-15-2001",
		"diag_name": "3",
		"diag_result": "6",
		"first_name": "7e0453536700a8d9f19f49441f1a1bab",
		"last_name": "68f3b3f9d7a7f21efee846b1a19cda3b",
		"mrn": "4d94e698dee0e06c55c995dc6f59ea55",
		"row_id": "ccb289c4-3aeb-4c27-87a1-1e7d18c94521"
	}
]
